app/helpers/works_helper.rb

```diff
@@ -112,6 +112,39 @@ def get_inspired_by(work)
     work.approved_related_works.where(translation: false)
   end
 
+  def related_work_note(related_work, relation, download: false)
+    work_link = link_to related_work.title, polymorphic_url(related_work)
+    language = tag.span(related_work.language.name, lang: related_work.language.short) if related_work.language
+    default_locale = download ? :en : nil
```

Comment:
Just to double check, because I'm having zero Google luck right now: when we let users choose a locale for the interface, any call to the `t` method including `locale: nil` should use the user's locale, not the site's default, right?

Reply:
I believe setting the locale only applies to mailers right now, but it looks like it will when we configure it to work with views. [source](https://github.com/ruby-i18n/i18n/blob/master/lib/i18n.rb#L211)